app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -325,11 +325,38 @@ def calculate_line_39
       end
 
       def is_ineligible_or_unsupported_for_property_tax_credit
-        StateFile::NjHomeownerEligibilityHelper.determine_eligibility(@intake) == StateFile::NjHomeownerEligibilityHelper::INELIGIBLE ||
-          Efile::Nj::NjPropertyTaxEligibility.ineligible?(@intake)
+        return true if Efile::Nj::NjPropertyTaxEligibility.ineligible?(@intake)
+
+        case @intake.household_rent_own
+        when "own"
+          StateFile::NjHomeownerEligibilityHelper.determine_eligibility(@intake) == StateFile::NjHomeownerEligibilityHelper::INELIGIBLE
+        when "rent"
+          StateFile::NjTenantEligibilityHelper.determine_eligibility(@intake) == StateFile::NjTenantEligibilityHelper::INELIGIBLE
+        when "both"
+          StateFile::NjTenantEligibilityHelper.determine_eligibility(@intake) == StateFile::NjTenantEligibilityHelper::INELIGIBLE &&
+            StateFile::NjHomeownerEligibilityHelper.determine_eligibility(@intake) == StateFile::NjHomeownerEligibilityHelper::INELIGIBLE
+        else
+          nil
+        end
```

Comment:
[boulder/sand] - discussed this with @jachan  on this PR https://github.com/codeforamerica/vita-min/pull/5178#discussion_r1884267898

since we have three utils for this: `NjHomeownerEligibilityHelper`, `NjTenantEligibilityHelper`, and `NjPropertyTaxEligibility`... we should consolidate the logic at some point and it being a sand on multiple PRs makes it harder for the next person, so sooner would be less debt.

We could start by moving `is_ineligible_or_unsupported_for_property_tax_credit()` out of the calculator, and then expanding the result types from `INELIGIBLE` to have more context.

Reply:
Captured in https://github.com/newjersey/affordability-pm/issues/227